### DISCUSSION TITLE
Can student use github copilot for free?

### DISCUSSION THREAD

**Discussion by rizkyhaksono** [community/community#30121] [Copilot Conversations]
With no subscription and no payment at all

**Comment by veverkap** [comment]
The GitHub Copilot subscription is available on a monthly or yearly cycle. If you choose a monthly billing cycle, you will be billed $10 per calendar month. If you choose a yearly billing cycle, you will be billed $100 per year. You can modify your billing cycle at any time, and the modification will be reflected from the start of your next billing cycle.

A free subscription for GitHub Copilot is available to verified students, and maintainers of popular open-source repositories on GitHub. If you meet the criteria as an open source maintainer, you will be automatically notified when you visit the GitHub Copilot subscription page. As a student, if you currently receive the GitHub Student Developer Pack, you will also be offered a free subscription when you visit the GitHub Copilot subscription page. For more information about the GitHub Student Developer Pack, see "[Apply to GitHub Global Campus as a student](https://docs.github.com/en/free-pro-team@latest/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)."

https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot#github-copilot-pricing
